### PR TITLE
Support skin attributes

### PR DESCRIPTION
**Pull request by fanzhanggoogle**
Added support for skinning attributes.
Added CesiumMan model.

### CODE REVIEW

File: examples/js/loaders/GLTFLoader.js [hunk 1]
```diff
@@ -481,6 +481,12 @@ THREE.GLTFLoader = ( function () {
 
 		var dracoLoader = this.dracoLoader;
 		var bufferViewIndex = primitive.extensions[ this.name ].bufferView;
+                // Add attribute id for skin attribute;
+                var attributesIdMap = primitive.extensions[ this.name ].attributes;
+                if (attributesIdMap['JOINTS_0'] !== undefined && attributesIdMap['WEIGHTS_0'] !== undefined) {
+                  dracoLoader.addAttributeIdForGenericAttributes(attributesIdMap['JOINTS_0'], 'skinIndex');
+                  dracoLoader.addAttributeIdForGenericAttributes(attributesIdMap['WEIGHTS_0'], 'skinWeight');
+                }
```
Comment (donmccurdy) [inline examples/js/loaders/GLTFLoader.js]: One other example to consider:

```js
if ( attributesIdMap['TEX_COORD_1'] !== undefined ) {
  dracoLoader.addAttributeIdForGenericAttributes( attributesIdMap[ 'TEX_COORD_1' ], 'uv2' );
}
```

**Comment by fanzhanggoogle** [comment]
Hi @donmccurdy , I have updated this PR to be consistent with the CL we discussed. That CL has been submitted already.

**Comment by donmccurdy** [comment]
Thanks @fanzhanggoogle! 👍 I'm aiming to re-open the three.js PR when the extension is submitted for ratification. Let me know if there are any other updates to include.

**Comment by fanzhanggoogle** [comment]
@donmccurdy Great! Thanks! I have also add the changes to your [drag-and-drop tool](https://github.com/fanzhanggoogle/three-gltf-viewer/tree/draco_extension). We could also update that when this change is merged to three.js.

**Pull request merged**